Composer/plugins/localPublish/src/index.ts

```diff
@@ -201,21 +223,23 @@ class LocalPublisher implements PublishPlugin<PublishConfig> {
     await this.zipBot(dstPath, srcDir);
   };
 
-  private saveSkillManifests = async (dstPath: string, srcDir: string) => {
-    const manifestSrcDir = this.getManifestSrcDir(srcDir);
-    const manifestDstDir = this.getManifestDstDir(dstPath);
+  private saveSkillManifests = async (dstPath: string, srcDir: string, runtimeType: string) => {
+    if (runtimeType === 'C#') {
```

Comment:
why would this only happen for C# bots?

Reply:
Yes, currently JS runtime and reference SDK not support manifest feature. I will implement it once the SDK support.